src/index.ts

```diff
@@ -96,6 +96,19 @@ task("starknet-compile", "Compiles Starknet contracts")
     )
     .setAction(starknetCompileAction);
 
+task("starknet-clean", "Deletes the content of the directory where compilation artifacts are stored")
+    .addOptionalVariadicPositionalParam("paths",
+        "The paths to be used for deployment.\n" +
+        "Each of the provided paths is recursively looked into while searching for compilation artifacts.\n" +
+        "If no paths are provided, the default contracts directory is traversed."
+    )
+    .addOptionalParam("cairoPath",
```

Comment:
The clean command doesn't need the cairoPath option, that's just something the compile task might need (rarely ever though).